### PR TITLE
Revise distribution charts

### PR DESCRIPTION
**Pull request by hancush**
this pr adds a composite bin for all salaries exceeding 200k to the distribution chart, so that wide distributions do not produce un-useful charts. closes #306.

### overall distribution
screen shot 2019-01-16 at 3 23 32 pm

### chicago
screen shot 2019-01-16 at 3 23 11 pm

### orland park
(no salaries above 200k)
screen shot 2019-01-16 at 3 22 39 pm

### CODE REVIEW

File: payroll/charts.py [hunk 1]
```diff
@@ -12,17 +10,13 @@ def _get_bar_color(self, lower_edge, upper_edge):
...
-        multiplier = 25000
-
-        # This is to make the appropriate number of bins
         max_value = np.amax(float_data)
-        bin_num = math.ceil(max_value / multiplier)  # rounding up to capture max value
-        bin_edges = np.array([], dtype='float')
 
-        for i in range(bin_num + 1):  # adding 1 to get appropriate number of bins
-            bin_edges = np.append(bin_edges, i * multiplier)
+        bin_size = 10000
```
Comment (hancush) [inline payroll/charts.py]: seems like one or both of these would be good candidates for `chart_settings`.

Comment (evz) [reply payroll/charts.py]: Any reason for not just sticking it in there?

Comment (hancush) [reply payroll/charts.py]: didn't feel strongly one way or the other, tbh. do you?

**Comment by hancush** [comment]
@fgregg – Made the composite bins striped.

Chicago:
screen shot 2019-01-18 at 9 48 30 am

Overall distribution:
screen shot 2019-01-18 at 9 48 15 am

**Comment by fgregg** [comment]
I think that's good. Not garish, but visually disruptive enough to not
mislead.

**Pull request merged**